NEWS

```diff
@@ -1,6 +1,6 @@
 4.3.0
 
-Add support for impression listener, there is an ability to use user-defined class to capture all impressions
+Add support for impression listener, there is an ability to use user-defined class to capture all impressions, add AttributeMatcher
```

Comment:
let's do one line per topic.
second line should be: "Now you can apply attribute matcher to the traffic type "